src/daemon/db_util.rs

```diff
@@ -168,15 +168,20 @@ where
 {
     let mut delegated_messages = vec![];
 
-    info!("Populating column EthMappings");
+    let hygge = state_manager.chain_config().epoch(Height::Hygge);
+    tracing::info!(
+        "Populating column EthMappings from range: [{}, {}]",
```

Comment:
Would it make sense to have it reversed for readability?

Reply:
Yeah, that might be clearer for the end user.